pocs/utils/error.py

```diff
@@ -1,28 +1,39 @@
 import sys
 
-from astropy.utils.exceptions import AstropyWarning
+from pocs.utils.logger import get_root_logger
 
-from pocs.base import PanBase
+logger = get_root_logger()
 
 
-class PanError(AstropyWarning, PanBase):
+class PanError(Exception):
 
     """ Base class for Panoptes errors """
 
     def __init__(self, msg=None, exit=False):
-        PanBase.__init__(self)
         if msg:
-            if exit:
-                self.exit_program(msg)
-            else:
-                self.logger.error('{}: {}'.format(self.__class__.__name__, msg))
-                self.msg = msg
+            logger.error('{}: {}'.format(self.__class__.__name__, msg))
 
-    def exit_program(self, msg='No reason specified'):
+        if exit:
+            self.exit_program(msg)
+
+        self.msg = msg
+
+    def exit_program(self, msg=None):
         """ Kills running program """
+        if not msg:
+            msg = 'No reason specified'
         print("TERMINATING: {}".format(msg))
         sys.exit(1)
 
+    def __str__(self):
+        error_str = 'PanError'
```

Comment:
Does `self.__class__.__name__` equal the derived class while in `__init__` of PanError? If so, it would be nice to have the name of the derived class used instead of PanError.